services/movie_session.py

```diff
@@ -0,0 +1,48 @@
+from db.models import MovieSession, CinemaHall, Movie
+
+
+def create_movie_session(movie_show_time: str,
+                         movie_id: int,
+                         cinema_hall_id: int,) -> None:
```

Comment:
The function `create_movie_session` is annotated to return `None`, but it actually returns a `MovieSession` instance. Consider changing the return type annotation to `MovieSession`.